demo/admin/src/common/MasterMenu.tsx

```diff
@@ -68,7 +70,20 @@ export const masterMenuData: MasterMenuData = [
                     <PagesPage
                         path={`/pages/pagetree/${match.params.category}`}
                         allCategories={pageTreeCategories}
-                        documentTypes={pageTreeDocumentTypes}
+                        documentTypes={(category): Record<DocumentType, DocumentInterface> => {
```

Comment:
For some reason, I get a very strange TypeScript error if I don't explicitly specify the return type here

Reply:
It would be nice if we could infer the return type.  What exactly is the TS error?